phase1/aws-ec2-terraform/phase2_implementations/simple-weave-single-master.yaml

```diff
@@ -0,0 +1,271 @@
+#cloud-config
```

Comment:
It's effectively the user-data file, which is effectively the point of hand-off.